test/LocationSpec.scala

```diff
@@ -0,0 +1,74 @@
+import org.specs2.mutable._
+
+import play.api.test._
+import play.api.test.Helpers._
+
+import utils.LocationFinder
+import utils.LocationFinder.{Location, Place}
+
+
+class LocationSpec extends Specification {
+
+  "Location" should {
+
+    "compute distance correctly" in {
+
+      "for zero distance" in {
+        val loc = Location(0, 0)
+        LocationFinder.dist(loc, loc) must be equalTo(0)
+      }
+
+      "for non-zero distance" in {
+        val loc1 = Location(0, 0)
+        val loc2 = Location(41.035232, -73.767067)
+        LocationFinder.dist(loc1, loc2) must be equalTo(84.4125016752632)
+      }
+    }
+
+    "get response from Places API" in {
+
+      "to find a location that exists" in new WithApplication {
+        val loc = Location(42.729913, -73.676565)
+        val placeName = "Rensselaer Union"
+
+        val place = LocationFinder.nearestPlacesLocation(loc).get
```

Comment:
I don't like doing it this way because these tests are dependent on the places API. If a new business opens up that's closer to this location then these tests fail. That said, I don't really know how to fix it, especially without changing the actual code in `LocationFinder`. I'm fine with these for now though; a solution is a shower-thought for a different day.